### ISSUE TITLE
Error when sending crash report

### ISSUE THREAD

**Issue by safouat** [GNS3/gns3-gui#3483]
Hello everyone, I'm new to GNS3. When I installed it, I encountered an issue where I couldn't access the 3080 port. Despite turning off my firewall, the problem persists. Do you have any suggestions on how to fix this bug?
I have tried the command "netstat -ano | findstr :3080"  to check if the port is being used by another process, but it give me any response. This indicates that there is no process currently using the port.
version:2.2.40.1
CPU:AMD RYZEN 5 5600U
![image](https://github.com/GNS3/gns3-gui/assets/120058233/8627f72b-0484-42a7-b10f-6f6e2f76974e)

**Comment by grossmj** [comment]
That's strange, do you use a proxy?

**Comment by safouat** [comment]
Thank you for your response. I resolved it  by switching my os from Windows 10 to Ubuntu, which resolved the problem and everything worked fine ..maybe the problem was from my proxy because changing os=>network configuration . However, I then encountered a separate issue with VMware where I received an error message stating "couldn't open dev/vmmon." I was able to resolve this problem by disabling secure boot from my BIOS. After taking this step, GNS3 worked properly.
